tests/e2e/cucumber/features/smoke/admin-settings/spaces.ocis.feature

```diff
@@ -93,33 +93,33 @@ Feature: spaces management
 
   Scenario: list members via sidebar
     Given "Admin" creates following users using API
-      | id      |
-      | Alice   |
-      | Brian   |
-      | Carol   |
-      | Marie   |
-      | Richard |
+      | id       |
+      | Alice    |
+      | Brian    |
+      | Carol    |
+      | Gehendra |
+      | Bishal   |
```

Comment:
```suggestion
      | id       |
      | Alice    |
      | Brian    |
      | Carol    |
      | Bob      |
      | Sam      |
```
may be give some western name. :smile:

Reply:
does it matter I gave nepali scientist name

Reply:
Maybe reconsider it. we are not the only ones writing the tests.